Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu:22.04
+
+WORKDIR /maki
+
+## Set up the base Ubuntu image
+RUN apt -y update
+RUN apt -y upgrade
+RUN apt install -y build-essential
+RUN apt-get update -qq -y
+RUN apt-get install -qq -y wget
+
+RUN apt install -y software-properties-common
+
+RUN apt install -y git
+
+# Get latest cmake
+RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
+RUN apt-add-repository 'deb https://apt.kitware.com/ubuntu/ jammy main'
+RUN apt update
+RUN apt install -y cmake
+
+
+RUN apt install -y ninja-build
+RUN apt install -y jq
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt install -y python3.10 python3-pip
+RUN python3 -m pip install -U numpy
```

Comment:
We don't need these Python packages for Maki (they were just for the artifact evaluation) so you can remove them